File: packages/coding-agent/test/suite/explore/fixtures/java/OrderService.java
/** Service that processes orders against inventory. */
public class OrderService {
	private final double taxRate;
	private int pending;

	public OrderService(double taxRate) {
		this.taxRate = taxRate;
		this.pending = 0;
	}

	/** Compute the total for an order including tax. */
	public double computeTotal(double amount) {
		return amount + amount * this.taxRate;
	}

	public void reset() {
		this.pending = 0;
	}
}

File: packages/coding-agent/test/suite/explore/fixtures/rs/order_service.rs
/// Service that processes orders against inventory.
pub struct OrderService {
    tax_rate: f64,
    pending: u32,
}

impl OrderService {
    /// Compute the total for an order including tax.
    pub fn compute_total(&self, amount: f64) -> f64 {
        amount + amount * self.tax_rate
    }

    pub fn reset(&mut self) {
        self.pending = 0;
    }
}
